### DISCUSSION TITLE
LTSP initramfs scripts

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#806] [Q&A] [answered]
I've been reading discussions about implementing LTSP in other distros and I have a question about initramfs:

did I understand that correctly, that the sole goal of LTSP scripts in initramfs is to overlay-mount squashfs image over NFS?
So if initramfs supports such mounting by itself, I don't need those scripts?

**Comment by alkisg** [comment, marked as answer]
You understood correctly.
For example, Raspberry Pi OS doesn't even use an initramfs, yet LTSP works fine there.

One thing you'll need is to add this in ltsp.conf, and then run `ltsp ipxe`:

```ini
[common]
KERNEL_PARAMETERS="init=/usr/share/ltsp/client/init/init"
```

For initramfs-tools based distributions such as Debian/Ubuntu, LTSP does that behind the scenes so it's not needed in the kernel command line.
